README.md

```diff
@@ -162,9 +162,14 @@ flowchart LR
 
 ```
 
+## Signals
 
+This app provides a single `wagtail_localize.signals.translation_imported`
+signal that is sent when translation are imported from Smartling.
 
-## Signals
+Signal kwargs:
 
-<!-- TODO -->
-- `translation_imported`
+- `sender`: The `wagtail_localize_smartling.models.Job` class
+- `instance`: The `Job` instance for which translation are being imported
+- `translation`: The `wagtail_localize.models.Translation` instance the translations are being imported to
+- `target_instance`: The model instance that `translation` is for (i.e. the instance returned by `translation.get_target_instance()`)
```

Comment:
I wonder if leaving just the `translation` kwarg is enough, with a note to "Use `translation.get_target_instance()` to get the model instance that the translation is for (e.g. a page or snippet)"

Reply:
Yeah, saves a query or two and leaves it up to the consumer. Good suggestion.

Reply:
Sorted in a96d87d